### DISCUSSION TITLE
Welcome to bladebit Discussions!

### DISCUSSION THREAD

**Discussion by harold-b** [Chia-Network/bladebit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Qman45** [comment]
Hi ...does this work with epyc processors?  Ive installed everything and entered bladebit, but it's not recognising it ... is it because of the epyc processors I'm using?

**Reply by harold-b** [reply]
Yes, 64-bit x86 based processor should have no problem with this. Many people have plotted already on a variety of EPYC processors.
